puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release_test.yaml

```diff
@@ -17,7 +17,7 @@
           repo_environment: staging
           pl_puppet: ''
           run_hammer_tests: false
-          db_type: mysql
+          db_type: ''
```

Comment:
We don't want to specify it as postgresql?

Reply:
Maybe the message wasn't totally correct, but I'd prefer to test with the defaults. This happens to be postgres. In a release test I think we want to test with defaults where possible.

Reply:
I think @komidore64 told us that we need to specify every parameter. That's why `pl_puppet` is there as well while being empty.